roles/calico/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+calico_image: ciscocloud/calico-node
+calico_image_tag: v0.4.8-mi
+
+bintray_baseurl: "https://bintray.com/artifact/download/loshkovskyi/generic"
```

Comment:
Yes, I already have such task in my todo list. If you give me access to the CiscoCloud bintray, I will update it shortly.